src/index.html

```diff
@@ -7,6 +7,574 @@
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#samsung"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#smartphones"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#laptops&computers"
+              data-qa="nav-hover"
+            >
+              Laptops & computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#gadgets"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#tablets"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#photo"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#video"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="page-main">
+      <div class="page-main__content content">
+        <div
+          class="content__item card"
+          data-qa="card"
+        >
+          <img
+            class="card__image"
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac"
+          >
+
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina
+            5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <div class="card__code">
+            <div class="card__code-text">
+              Product code:&nbsp;
+            </div>
+
+            <div class="card__code-value">
+              195434
+            </div>
+          </div>
+
+          <div class="card__rating">
+            <div class="card__stars stars stars--4">
+              <div class="stars__star star"></div>
+              <div class="stars__star star"></div>
+              <div class="stars__star star"></div>
+              <div class="stars__star star"></div>
+              <div class="stars__star star"></div>
+            </div>
+
+            <div class="card__reviews">
+              <div class="card__reviews-text">
+                Reviews:&nbsp;
+              </div>
+
+              <div class="card__reviews-value">
+                5
+              </div>
+            </div>
+          </div>
+
+          <div class="card__price">
+            <div class="card__price-text">
+              Price:
+            </div>
+
+            <div class="card__price-value">
+              $2,199
+            </div>
```

Comment:
Do not use for text content, use <p> instead. Check all places

Reply:
done